packages/fxa-auth-server/lib/senders/templates/passwordChangeRequired.txt

```diff
@@ -1,10 +1,12 @@
+{{t "Password Change Required"}}
+
 {{{t "We detected suspicious behavior on your Firefox account. To prevent unauthorized access to your Firefox Account, we’ve disconnected all devices on your account and are requiring you to change your password as a precaution." }}}
 
 {{{t "Sign back into any device or service where you use your Firefox account and follow the steps that will be presented to you." }}}
 
-{{{t "Important: pick a different password than what you were previously using and make sure that it is different from your email account." }}}
+{{{t "Important: Pick a different password than what you were previously using and make sure that it is different from your email account." }}}
 
 {{{t "Best," }}}
-{{{t "The Firefox Accounts team" }}}
+{{{t "the Firefox Accounts team" }}}
```

Comment:
Oh whoops, I see I noted in the issue that "the" should be lowercase, but I didn't realize it was at the beginning of the sentence. These "the"s you changed can be reverted back to "The"s.